README.md

```diff
@@ -1,8 +1,17 @@
+# Eclipse LSP4MP - Language Server for Eclipse MicroProfile
+
 [![Build Status](https://ci.eclipse.org/lsp4mp/buildStatus/icon?job=lsp4mp%2Fmaster)](https://ci.eclipse.org/lsp4mp/job/lsp4mp/job/master/)
+[![License](https://img.shields.io/badge/License-EPL%202.0-brightgreen.svg)](https://www.eclipse.org/legal/epl-2.0/)
+
+A language server implementation based on the [Language Server Protocol](https://github.com/Microsoft/language-server-protocol) for [Eclipse MicroProfile](https://microprofile.io/). This Language Server for Eclipse MicroProfile (LSP4MP) provides core language support capabilities (such as code complete, diagnostics, quick fixes) to enable developers to quickly and easily develop applications using MicroProfile APIs.

```

Comment:
Your information is very good but I think we should explain more what features are provided (properties, java and yaml support like https://github.com/redhat-developer/vscode-quarkus#quarkusmicroprofile-properties-features) and mention that features like completion depends on the Java project dependencies. 

It will explain why we need MP JDT  LS extension to provide MP support according dependencies of the Java project of the user.

After that I think we should mention too that MicroProfile JDT LS Extensions and MP LS are extensible:

 * for MicroProfile JDT LS Extensions extensible by developping an Eclipse plugin and MP extensions points to provide for instance custom properties and add a ling to quarkus-ls to give a sample of extension with Quarkus which provides properties by using Quarkus deployment mechanism.

 * for MicroProfile LS extensible to provide custom snippets for java and properties files.